### ISSUE TITLE
Custom Tags per TaggedItemBase not connected to objects in admin/taggit/ 

### ISSUE THREAD

**Issue by Braintelligence** [alex/django-taggit#376]
Hello everyone.

So I created a model with Custom Tags:
```python
class ArticleTaggedItemBase(TaggedItemBase):
    content_object = models.ForeignKey('Article')

class Article(models.Model):
    title = models.CharField(max_length=300)
    comment = models.CharField(max_length=600, blank=True)

    tags = TaggableManager(through=ArticleTaggedItemBase)
```

Works fine from what I understand but the problem is that you can't see the tags in /admin/taggit:
![image](https://cloud.githubusercontent.com/assets/6761196/12689686/be597b08-c6de-11e5-8697-db84f3d82e70.png)
![image](https://cloud.githubusercontent.com/assets/6761196/12689704/dcaf9a56-c6de-11e5-8f94-b43409b91d81.png)

So as you can see the tags show up fine within the tagged model object but the connection doesn't show up in the /admin/taggit view.
I guess that Custom Tags don't work with that view properly?

If so, and if there's no simple way to correct this, how do I turn off the /admin/taggit link altogether?

**Comment by alexadkins** [comment]
I'm having the same issue.

**Comment by chirdeeptomar** [comment]
Same issue if primary keys are UUIDs. django-taggit creates a new table in the database which I am not happy about. It does it as I created a class derived from GenericUUIDTaggedItemBase in my django app. Taggit should have a setting if your primary keys are UUID.

**Comment by rtpg** [comment]
Yep, as people have identified this is due to custom tag models using different tables etc. Seems very doable for us to improve this (or at least document some workarounds)

**Comment by ggiller-collective** [comment]
Any update on this issue @rtpg?

**Comment by rtpg** [comment]
@ggiller-collective no progress has been made on this specific point.

Generally if you make a custom tag, you should be able to add that model to the admin like any other model (generally in some `admin.py` file, see the Django docs)

As to `admin/taggit` appearing at all in that case.... I have not experimented with this, but you might be able to use `AdminSite.unregister` to remove `taggit.models.TaggedItemBase` or otherwise remove `admin/taggit` entirely from the admin views.  We might want to add an option to do this or something.

This is all working "as designed", in that derived models won't share the same admin pages as the base models because they're not the same model! But we can clearly make improvements given people coming here and asking this
